.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Please note that GitHub issues are meant to report bugs or feature requests, and are not intended for "How to" questions or general support. GitHub issues requesting such support will be closed. Luckily, we have an active community on the GIS StackExchange [1] and the PDAL Mailing List [2] that are more than happy to chime in with advice in these situations!
+
+[1] https://gis.stackexchange.com/questions/tagged/pdal
+[2] https://pdal.io/community.html#mailing-list
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
Think this should be merged with the above.  What I want 99% of the time is the exact command someone has run along with a pipeline file if that's relevant.

Reply:
Agreed, and I had actually intended to ask for the pipeline too, just forgot.